src/Profile.js

```diff
@@ -49,12 +54,18 @@ const Box = styled.div`
     height: 0;
     border: 8px solid black;
     top: -8px;
-    right: 20px;
+    /* width / 3 = width of img */
+    /* width / 3 - 16px (self-width) - 2px (margin) */
+    right: calc(33.33% / 2 - 14px);
     border-width: 0 8px 8px 8px;
     border-color: transparent transparent #ccc transparent;
   }
 `;
 
+const P = styled.p`
```

Comment:
not sure why but html tag names for styled components field weird to me. But it should not be a problem

Reply:
Ah I see, no problem calling it "Name" or sth.

Reply:
Fixed 👍